adlik_serving/runtime/provider/grpc_predict_request_provider.cc

```diff
@@ -38,34 +33,20 @@ tensorflow::Status GRPCPredictRequestProvider::create(const ModelId& id,
     }
   }
 
-  provider->reset(new GRPCPredictRequestProvider(id, request));
+  provider->reset(new GRPCPredictRequestProvider(request));
```

Comment:
Avoid using `new` and `delete` directly, use `*provider = std::make_unique<GRPCPredictRequestProvider>(request)` instead;